### PR TITLE
added ActorSelection benchmarks per #1555

### PR DESCRIPTION
**Pull request by Aaronontheweb**
Still working on the Ask benchmarks and stress tests

### CODE REVIEW

File: src/contrib/cluster/Akka.Cluster.Sharding.Tests/Akka.Cluster.Sharding.Tests.csproj [hunk 1]
```diff
@@ -124,6 +124,9 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```
Comment (rogeralsing) [inline src/contrib/cluster/Akka.Cluster.Sharding.Tests/Akka.Cluster.Sharding.Tests.csproj]: What is this line?

File: src/core/Akka.Tests.Performance/Actor/ActorSelectionSpecs.cs [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                ++i;
+            }
+            _resetEvent.Wait();
+        }
+
+        [PerfBenchmark(Description = "Tests the message delivery throughput of NEW ActorSelections to NEW actors. This is really a stress test.",
+            NumberOfIterations = 13, RunMode = RunMode.Iterations, TestMode = TestMode.Measurement)]
+        [CounterMeasurement(ActorSelectionCounterName)]
+        public void NewActorSelectionOnPreExistingActorThroughput(BenchmarkContext context)
+        {
```
Comment (rogeralsing) [inline src/core/Akka.Tests.Performance/Actor/ActorSelectionSpecs.cs]: How does this test differ from ReusedActorSelectionOnPreExistingActorThroughput ?
They both contain the same line `var actorSelection = System.ActorSelection(_receiverActorPath);`

File: src/contrib/cluster/Akka.Cluster.Sharding.Tests/Akka.Cluster.Sharding.Tests.csproj (hunk 1 above)
Comment (Aaronontheweb) [inline src/contrib/cluster/Akka.Cluster.Sharding.Tests/Akka.Cluster.Sharding.Tests.csproj]: I have no idea TBH - it's shown up on multiple PRs. Any ideas @Horusiath ?

File: src/core/Akka.Tests.Performance/Actor/ActorSelectionSpecs.cs (hunk 2 above)
Comment (Aaronontheweb) [inline src/core/Akka.Tests.Performance/Actor/ActorSelectionSpecs.cs]: I'll check - might have made a mistake.

Comment (Aaronontheweb) [inline src/core/Akka.Tests.Performance/Actor/ActorSelectionSpecs.cs]: Yep, I did - I goofed. should be using a new `ActorSelection` each time.

**Comment by Aaronontheweb** [comment]
@rogeralsing ok, addressed those issues.

**Pull request merged**